### ISSUE TITLE
Disk dumper

### ISSUE THREAD

**Issue by mariogamer2** [dimok789/loadiine_gx2#64]
Like USB loader GX,when inserting a disk,it will be good to be able to dump it.

**Comment by Maschell** [comment]
At the moment we only have acces to the content folder of a disc when were in the game, so its not possible yet. Maybe this will be possible with an IOSU exploit?

**Comment by MSco** [comment]
But how does ddd get access to /vol/code, /vol/meta, /vol/save then? Shouldnt loadiine have the same permissions as ddd with the kernel exploit? Thus, it should be possible to implement a dump feature in loadiine.

**Comment by CreeperMario** [comment]
Fair enough, but why not just use DDD? DDD can be launched within the Homebrew Launcher.
@dimok789 probably has a good reason for having DDD separated from Loadiine GX2.

**Comment by dibas** [comment]
I'm closing this issue for now. This issue will be reopened when/if anyone wants to implement it. I know from n1ghty that we do have a way of temporarily saving files, so it should be already theoretically possible now, but directly having access to /vol/ would simplify implementing this by a lot.
